pkgs/development/python-modules/dicom2nifti/default.nix

```diff
@@ -39,24 +39,39 @@ buildPythonPackage rec {
     scipy
```

Comment:
I'm not familiar with `propagatedBuildInputs` in `buildPythonPackage`, are these the program's dependencies?

Reply:
I've spotted them in both `setup.py` (under `install_requires`) and `requirements.txt`; 
It seems to me these belong in `dependencies = [ ... ];`

Reply:
yeah and IIUC they are propagated to the runtime environment. because the package needs them to run

Reply:
your second comment didnt load when i sent my comment. sorry.

So tell me if im wrong please. The reason you asked for the runtime dependencies to be seperated out to dependencies attribute is because the builder doesnt need to have them during build time. And this would allow us to not propagate `buildInput` related stuff to the nixpkgs users because they wont be building the package only will be using it. Am i right?

Reply:
[This comment](https://github.com/NixOS/nixpkgs/blob/171b47803df9f2340b73aadd18e7f749a461dab0/pkgs/development/interpreters/python/mk-python-derivation.nix#L136) on the `buildPythonPackage` definition states that `propagatedBuildInputs` is used for dependencies that are to be importable by programs that depend on `dicom2nifti`.

As the README for `dicom2nifti` does not include any examples where its dependencies are imported, I don't see a reason to make these available.

Reply:
[Directly underneath](https://github.com/NixOS/nixpkgs/blob/171b47803df9f2340b73aadd18e7f749a461dab0/pkgs/development/interpreters/python/mk-python-derivation.nix#L142) is also an explanation for when to use `dependencies`.

Ctrl + F'ing your way the same file will lead you to [this definition](https://github.com/NixOS/nixpkgs/blob/171b47803df9f2340b73aadd18e7f749a461dab0/pkgs/development/interpreters/python/mk-python-derivation.nix#L368), where you see that `dependencies` is joined with `propagatedBuildInputs`, meaning they are indeed both visible during the runtime.